shell/lisa_shell

```diff
@@ -449,7 +457,11 @@ $(lisa-version)
 Type "lisa-help" for on-line help on available commands
 EOF
 
-# Setup default SHELL text color
-echo -e "$LISASHELL_DEFAULT"
-
+    # Setup default SHELL text color
+    echo -e "$LISASHELL_DEFAULT"
+else
+    lisa-version
+    echo "Environment variables:"
+    printenv | grep --color=no LISA
```

Comment:
:heart:

Reply:
Yeah, I stole the idea from AOSP's lunch thingy. But I just figured, this is in fact half broken. We export more stuff than just `LISA_` variables. We also have stuff for exekall and wltests. So maybe I should try and do a diff on `printenv` before/after sourcing init_env or something ...

Reply:
That could work, but this is already a good step forward anyway.

Reply:
OK, I'll keep things simple for now then. We can always extend later if need be.

Reply:
For exekall and bisector, env var are only used as defaults values, so it's probably better to use the CLI option instead of updating these in 3rd party scripts. (one exception are Artifactorial token and folder).

A plain diff will also show things that should not be modified related to the venv, so IMO it's better to have an explicit grep like you did. It can be expanded to cover bisector, exekall and wltest.